app/src/main/java/com/nervousfish/nervousfish/activities/LoginActivity.java

```diff
@@ -49,6 +48,7 @@ public void onClick(final View v) {
         });
     }
 
+    @SuppressWarnings("PMD.AvoidFinalLocalVariable")
```

Comment:
Yes, because the dummypassword is there and you remove it in another pr. So it's not needed to remove dummypassword in this pr